crates/core/tedge/src/cli/certificate/create.rs

```diff
@@ -76,16 +71,25 @@ impl CreateCertCmd {
     }
 }
 
-fn create_new_file(path: impl AsRef<Path>) -> Result<File, CertError> {
-    Ok(OpenOptions::new().write(true).create_new(true).open(path)?)
+fn create_new_file(path: impl AsRef<Path>, user: &str, group: &str) -> Result<File, CertError> {
+    let file = OpenOptions::new()
+        .write(true)
+        .create_new(true)
+        .open(path.as_ref())?;
```

Comment:
It is not necessary to call `as_ref()` here, as [OpenOptions::open() takes an `AsRef<Path>`](https://doc.rust-lang.org/stable/std/fs/struct.OpenOptions.html#method.open), so you can pass `path` directly, I believe! :+1:

Reply:
> Hi!
> 
> One question: That would make the `tedge_users` crate unused in the codebase, right? So how about `git rm`ing it altogether in this PR?

Oops. I did it; but forgot to push the commit.

Reply:
> It is not necessary to call `as_ref()` here, as [OpenOptions::open() takes an `AsRef<Path>`](https://doc.rust-lang.org/stable/std/fs/struct.OpenOptions.html#method.open), so you can pass `path` directly, I believe! 👍



> It is not necessary to call `as_ref()` here, as [OpenOptions::open() takes an `AsRef<Path>`](https://doc.rust-lang.org/stable/std/fs/struct.OpenOptions.html#method.open), so you can pass `path` directly, I believe! 👍

This to avoid giving ownership of the `path` to the `open` function. This `path` is used twice.

Reply:
Ah, you're totally right! I missed the second usage!